scripts/linux/buildAPIProxy.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+#https://github.com/Azure/iotedge/blob/iiot/scripts/linux/cross-platform-rust-build.sh
+###############################################################################
+# This script builds a static binary of the api-proxy-module
+###############################################################################
+
+set -e
+
+###############################################################################
+# Define Environment Variables
+###############################################################################
+# Get directory of running script
+ARCH=$(uname -m)
+DIR=$(cd "$(dirname "$0")" && pwd)
+TARGET=
+STRIP=
+SCRIPT_NAME=$(basename "$0")
+PROJECT=
+DOCKERFILE=
+DOCKER_IMAGENAME=
+DEFAULT_DOCKER_NAMESPACE="microsoft"
+DOCKER_NAMESPACE=${DEFAULT_DOCKER_NAMESPACE}
```

Comment:
I can't find below script using DOCKER_NAMESPACE variable.

Reply:
I can remove it but again it will make the merge between buildAPIProxy.sh and buildimage.sh more difficult.